sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -159,3 +163,110 @@ def __init__(self, history, *args, **kwargs):
         self.history = history
         message = "Reached maximum redirect attempts."
         super(TooManyRedirectsError, self).__init__(message, *args, **kwargs)
+
+class ODataV4Format(object):
```

Comment:
I'm a bit concerned that we potentially lose information in deserialization. We've had to go back and plump that in in other scenarios (e.g. `additional_properties`)

Reply:
@johanste You can get the original json with `exp.odata_json` , is this enough?